qiskit/transpiler/passes/layout/vf2_post_layout.py

```diff
@@ -153,16 +142,12 @@ def __init__(
 
     def run(self, dag):
         """run the layout method"""
-        if self.target is None and (self.coupling_map is None or self.properties is None):
-            raise TranspilerError(
-                "A target must be specified or a coupling map and properties must be provided"
-            )
+        if self.target is None:
+            raise TranspilerError("A target must be specified or a coupling map must be provided")
```

Comment:
This threw me for a second until I remember these passes should be instantiable even if the required target isn't specified.

Reply:
Yes I also spent a long time thinking if I should make it a required argument.